SSToastMessage/Sources/Classes/MessageView.swift

```diff
@@ -51,6 +51,9 @@ public struct MessageView<MessageContent>: ViewModifier where MessageContent: Vi
     /// Allow to perform action on tap default is
     var onTap: () -> Void
     
+    /// Allow to perform any action when toast hide
+    var onToastHide: () -> Void
```

Comment:
Suggestion: This naming convention aligns with existing functions, yet for clarity, it would be preferable to rename `onToastHide` as `onDismiss` or a variant incorporating 'dismiss' for better semantic coherence.